code/modules/projectiles/targeting.dm

```diff
@@ -62,6 +62,8 @@
 
 //Aiming at the target mob.
 /obj/item/weapon/gun/proc/Aim(var/mob/living/M)
+	if (M.alpha <= 1)
+		return // no abusing aiming to reveal cloaked individuals. Why do cloaking cloaks set alpha to specifically 1 anyway? The human eye can't see that 'cept maybe on a fully white background.
```

Comment:
Probably so it can still be right-clickable

Reply:
Doesn't seem so 
![image](https://user-images.githubusercontent.com/7573912/129423381-dad06637-f467-4aba-8b4c-974ec3cdabc0.png)

Reply:
Hmm..........